docs/advanced/dcrtime.md

```diff
@@ -0,0 +1,105 @@
+# <img class="dcr-icon" src="/img/dcr-icons/Servers.svg" /> Dcrtime
+
+---
+
+Dcrtime is a timestamping application that allows users to store hashes of arbitrary data on the Decred blockchain. Inspired by Peter Todd's [opentimestamps](https://petertodd.org/2016/opentimestamps-announcement), dcrtime was originally built to store timestamped hashes of data from [Politeia](../../governance/politeia/overview), Decred's proposal system, on the Decred blockchain. 
```

Comment:
OpenTimestamps caps.

Reply:
Link to `overview.md` file

Reply:
Done

Reply:
answering your question below, the correct path for the link should be `../governance/politeia/overview.md`

Reply:
Fixed